### PR TITLE
[TEST] SMP restore

### PR DESCRIPTION
**Pull request by lyakh**
test https://github.com/zephyrproject-rtos/zephyr/pull/68366

### CODE REVIEW

**Comment by marc-hb** [comment]
CAVS NOCODEC configurations fails most tests in https://sof-ci.01.org/sofpr/PR8823/build2486/devicetest/index.html with an IPC timeout.

Same in ACE NOCODEC https://sof-ci.01.org/sofpr/PR8823/build2485/devicetest/index.html

Nothing in mtrace unfortunately: silent crash.

"CODEC" configurations are passing besides the usual suspend/resume failures.

File: west.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: 9852e8e15bc8536aa1a49cc2697c1e8f802e331f
+      revision: pull/68366/head
```
Comment (marc-hb) [inline west.yml:46]: - https://github.com/zephyrproject-rtos/zephyr/pull/68366

**Comment by marc-hb** [comment]
Similarly bad NOCODEC results in variant #8824

**Comment by lyakh** [comment]
interesting. Comparing CI results to #8804: not very clear. The MTL failure there https://sof-ci.01.org/sofpr/PR8804/build2377/devicetest/index.html is similar to this PR's https://sof-ci.01.org/sofpr/PR8823/build2511/devicetest/index.html but their TGL failure https://sof-ci.01.org/sofpr/PR8804/build2378/devicetest/index.html is gone in our https://sof-ci.01.org/sofpr/PR8823/build2512/devicetest/index.html

**Pull request closed without merging**